terraform/terraform.tfvars.example

```diff
@@ -35,3 +35,12 @@ mgmt_dns_support = "<true or false>"
 
 # Type: string, it's a DNS zone name so don't get crazy
 private_mgmt_zone_name = "<dns zone name>"
+
+# Type: string, just a name for the log group name for the MGMT VPC
+mgmt_flow_log_group_name = "<name_of_a_log_group>"
+
+# Type: string, just a name for the IAM role that will be attached to the VPC logging
+devsecops_iam_log_role_name = "<name_for_iam_log_role"
+
+$ Type: string, just a name for the log policy to attach to the VPCs
+devsecops_flow_log_policy = "<name_for_flow_log_policy"
```

Comment:
My style suggestions are:

* Moving the text to the [`description`](https://www.terraform.io/docs/configuration/variables.html#description-1) of each variable
* Only leaving the required variables in this one

All good for this pull request since it's consistent with the rest of the file.